base/meta.jl

```diff
@@ -12,15 +12,64 @@ export quot,
        show_sexpr,
        @dump
 
+"""
+    quot(ex)::Expr
+
+Quote expression `ex` to produce an expression with head `quote`. This can for instance be used to represent objects of type `Expr` in the AST.
+```jldoctest
+julia> eval(quot(:x))
+:x
+
+julia> dump(quot(:x))
+Expr
+  head: Symbol quote
+  args: Array{Any}((1,))
+    1: Symbol x
+
+julia> eval(quot(:(1+2)))
+:(1 + 2)
+```
+See also the manual section about [QuoteNode](@ref man-quote-node).
```

Comment:
Maybe move this to above the examples? I think thats what we usually do

Reply:
ok